### PR TITLE
ref(Image): from JS to TS and add tests

### PR DESCRIPTION
**Pull request by NarekArshakyan**

### CODE REVIEW

File: src/lib/atoms/Image/Image.stories.tsx [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+import Button from '../Button';
 
-export default {
+// Types
+import { IImageProps } from '.';
+
+const meta: Meta<typeof ImageComponent> = {
     title: 'Atoms/Image',
     component: ImageComponent,
     argTypes: {
```
Comment (GaroGabrielyan) [inline src/lib/atoms/Image/Image.stories.tsx:15]: Change all `category: category....` to `...propCategory.`

File: src/lib/atoms/Image/Image.stories.tsx [hunk 2]
```diff
@@ -26,18 +31,24 @@ export default {
...
 };
 
-const Template = ({ ...args }) => <ImageComponent {...args} />;
+const Template: FC<IImageProps> = ({ ...args }) => <ImageComponent {...args} />;
+
+export default meta;
 
 export const Default = Template.bind({});
+
 Default.args = {
```
Comment (GaroGabrielyan) [inline src/lib/atoms/Image/Image.stories.tsx:40]: Please check all default props and don't repeat.

File: src/lib/atoms/Image/Image.tsx [hunk 3]
```diff
@@ -8,25 +7,73 @@ import Tooltip from '../../molecules/Tooltip';
...
+    /**
+     * Will add a title to the top of Image atom. Any valid React node
+     */
+    title: string;
+    /**
+     * Title for 'Tooltip'.
+     */
+    tooltipTitle: string | ReactNode;
+    /**
+     * Addional className
```
Comment (GaroGabrielyan) [inline src/lib/atoms/Image/Image.tsx:46]: Typo: In word 'Addional'

File: src/lib/atoms/Image/Image.tsx [hunk 4]
```diff
@@ -8,25 +7,73 @@ import Tooltip from '../../molecules/Tooltip';
...
     className,
-    isValid,
+    isValid = true,
     tooltipTitle,
-    emptyText,
+    emptyText = 'No image to display',
     ...restProps
-}) {
+}) => {
     const isValidSource = useMemo(() => typeof src === 'string' && src !== '', [src]);
```
Comment (GaroGabrielyan) [inline src/lib/atoms/Image/Image.tsx:73]:
```suggestion
    const isValidSource = useMemo(() => typeof src === 'string' && src.length, [src]);
```

File: src/lib/atoms/Image/Image.tsx [hunk 5]
```diff
@@ -8,25 +7,73 @@ import Tooltip from '../../molecules/Tooltip';
...
+     * Image atom also can be included in "Form" organism. It can be validated as other "Form" elements
+     */
+    isValid: boolean;
+    /**
+     * Empty state text for component
+     */
+    emptyText: string;
+}
+
+const Image: FC<Partial<IImageProps>> = ({
```
Comment (GaroGabrielyan) [inline src/lib/atoms/Image/Image.tsx:59]: Please remove Partial and in storybook Image Docs you can see all types descriptions.
```suggestion
const Image: FC<IImageProps> = ({
```

**Review by GaroGabrielyan — CHANGES_REQUESTED** [review]

File: src/lib/atoms/Image/Image.tsx (hunk 4 above)
Comment (NarekArshakyan) [reply src/lib/atoms/Image/Image.tsx:73]: these are the same expression

**Review by GaroGabrielyan — APPROVED** [review]

File: src/lib/atoms/Image/Image.test.tsx [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+            }
+        );
+    });
+
+    it('renders without crashing', () => {
+        expect(setup.exists()).toBeTruthy();
+    });
+
+    it('renders className prop correctly', () => {
+        const className = 'test';
```
Comment (NairaMisakyan) [inline src/lib/atoms/Image/Image.test.tsx:38]: className is declared but its value is never read

**Pull request merged**